playbooks/openedx_native.yml

```diff
@@ -110,3 +110,5 @@
       when: not COMMON_ENABLE_DATADOG
     - role: user_retirement_pipeline
       when: COMMON_RETIREMENT_SERVICE_SETUP
+    - role: tinymce_plugins
+      when: (TINYMCE_ADDITIONAL_PLUGINS is defined) and (TINYMCE_ADDITIONAL_PLUGINS|length > 0)
```

Comment:
@nizarmah Can we use a different variable name here (something like `TINYMCE_ADDITIONAL_PLUGINS_LIST`)? We are using the same name inside the `ADDITIONAL_NODE_ENV_VARS` but values defined in each are different. It could get confusing.

Reply:
I made that change :+1: Thanks for suggesting an alternative by the way!